torch/ao/quantization/quantization_mappings.py

```diff
@@ -128,11 +128,11 @@
 # Default mapping from floating point function or torch ops to quantized ops
 # TODO: merge with default static mapping
 DEFAULT_FLOAT_TO_QUANTIZED_OPERATOR_MAPPINGS : Dict[Union[Callable, str], Callable] = {
-    F.elu: torch._ops.ops.quantized.elu,
-    F.hardswish: torch._ops.ops.quantized.hardswish,
-    F.instance_norm: torch._ops.ops.quantized.instance_norm,
-    F.layer_norm: torch._ops.ops.quantized.layer_norm,
-    F.leaky_relu: torch._ops.ops.quantized.leaky_relu,
+    F.elu: torch.ops.quantized.elu,
```

Comment:
I added a comment before but can't see it for some reason. This is not a necessary change but I don't see a reason why we need to get the ops from `torch._ops.ops` instead of `torch.ops`. I can undo it if there are other implications of this change

Reply:
Ho ok. To avoid confusion, these unrelated refactor can be done in a separate PR as well.